fusor-ember-cli/app/controllers/register-nodes.js

```diff
@@ -1,7 +1,6 @@
 import Ember from 'ember';
-import DeploymentControllerMixin from "../mixins/deployment-controller-mixin";
```

Comment:
`DeploymentControllerMixin` is a mix to DRY things up for controllers `deployment` and `deployment-new`